libr/bin/format/nes/nes_specs.h

```diff
@@ -61,7 +61,11 @@
 #define JOYPAD_PORT1                        0x4016
 #define JOYPAD_PORT2                        0x4017
 
+#ifdef _MSC_VER
+typedef struct {
+#else
 typedef struct __attribute__((__packed__)) {
```

Comment:
we should add in rtypesbase.h a define for this attribute to be portable across compilers. how to do this with msvc?